Wireframe/my_work.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Purpose of a wireframe, purpose of a readme file, branching in GIT.</title>
+        <link rel="stylesheet" href="my_style.css" />
+    </head>
+
+    <body>
+        <header>
+            <h1>WIREFRAME</h1>
+            <p>A wireframe, or website wireframe, also known as page schematic, or screen blueprint, is a visual guide that represent the skeletal framework of a website.
+                In this article, we describe the purpose of a wireframe and a Read.me file, and explore what is a branch in GIT.
+            </p>
+        </header>
+        <main>
+            <article>
+                <img src="my_wireframe_pic.png" alt="wireframe_sample" />
+                <h2>Purpose of a Wireframe</h2>
+                <p>
+                    A wireframe is a visual diagram that outlines the structure and layout of a website, app, or other digital product. The purpose of a wireframe is to:
+                </p>
+                <ul>
+                    <li><b>Plan the Layout:</b><br>
```

Comment:
There is a better semantic tag for bolded/important text. Please, try to find yourself which tag.